dotcom-rendering/src/web/lib/edition.ts

```diff
@@ -0,0 +1,36 @@
+const UkEdition: Edition = {
+	id: 'UK',
+	displayName: 'UK Edition',
+	locale: 'en-gb',
+};
+
+const UsEdition: Edition = {
+	id: 'US',
+	displayName: 'US Edition',
+	locale: 'en-us',
+};
+
+const AuEdition: Edition = {
+	id: 'AU',
+	displayName: 'Australia edition',
+	locale: 'en-au',
+};
+
+const InternationalEdition: Edition = {
+	id: 'INT',
+	displayName: 'International edition',
+	locale: 'en-gb',
+};
+
+export const getEditionFromId = (id: EditionId): Edition => {
```

Comment:
Are we anticipating using this file, method, contents, etc only in the one location it is currently used? If not, we could simplify this to something like 

```
export const getEditionFromId = (id: EditionId): Edition => {
switch (id) {
		case 'US':
			return 'en-us';
		case 'AU':
			return 'en-au';
		default:
			return 'en-gb';
	}}
```

Reply:
I would expect that this will be a starting point for migrating date strings from Frontend to DCR, and that we might see this file used more often in the future!